beacon_node/execution_layer/src/lib.rs

```diff
@@ -551,6 +565,50 @@ impl ExecutionLayer {
         )
     }
 
+    pub async fn exchange_transition_configuration(&self, spec: &ChainSpec) {
+        let local = TransitionConfigurationV1 {
+            terminal_total_difficulty: spec.terminal_total_difficulty,
+            terminal_block_hash: spec.terminal_block_hash,
+            // TODO(paul): confirm that we don't know this value.
```

Comment:
Can we delete this comment now that it's confirmed?

Reply:
Yep, thanks!